<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login - Mattermost</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/icons/img/icon0.ico</normaloff>:/icons/img/icon0.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="icon">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Login to Mattermost</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,5">
     <item row="0" column="0">
      <widget class="QLabel" name="domain_label">
       <property name="toolTip">
        <string>The Mattermost server you are connecting to</string>
       </property>
       <property name="text">
        <string>&amp;Domain</string>
       </property>
       <property name="buddy">
        <cstring>domain_lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="username_lineEdit">
       <property name="maxLength">
        <number>512</number>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="username_label">
       <property name="toolTip">
        <string>Your username</string>
       </property>
       <property name="text">
        <string>&amp;Username</string>
       </property>
       <property name="buddy">
        <cstring>username_lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="password_label">
       <property name="toolTip">
        <string>Your password</string>
       </property>
       <property name="text">
        <string>&amp;Password</string>
       </property>
       <property name="buddy">
        <cstring>password_lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="domain_lineEdit">
       <property name="toolTip">
        <string/>
       </property>
       <property name="maxLength">
        <number>512</number>
       </property>
       <property name="placeholderText">
        <string>Domain</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="maxLength">
        <number>512</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Some servers uses SSO auth. This client doesn't support it natevly.
Instead you should bypass SSO auth into your browser, find 'MMAUTHTOKEN'
cookie in developer tools and copy its content into field below.
Password and token can't be used simultenusly</string>
     </property>
     <property name="text">
      <string>Token authentication</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,5">
     <item row="0" column="1">
      <widget class="QLineEdit" name="token_lineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Token (optional)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="token_label">
       <property name="toolTip">
        <string>Token that should be used for authentication instead of password</string>
       </property>
       <property name="text">
        <string>&amp;Token</string>
       </property>
       <property name="buddy">
        <cstring>token_lineEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="error_label">
     <property name="styleSheet">
      <string notr="true">color: red</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>83</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="loginInProgressLabel">
       <property name="text">
        <string>Logging in...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="login_pushButton">
       <property name="text">
        <string>&amp;Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>domain_lineEdit</tabstop>
  <tabstop>username_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>login_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
